### PR TITLE
Release 3.7.0

### PR DESCRIPTION
**Pull request by fjorgemota**
### Changes proposed in this Pull Request

- Update translations with translator comment to a single line in https://github.com/Automattic/syntaxhighlighter/pull/245
- Fix nested shortcode bugs in https://github.com/Automattic/syntaxhighlighter/pull/261
- PHP 8 Compat: declare class properties in https://github.com/Automattic/syntaxhighlighter/pull/265
- Migrate project to wp-scripts and fix lodash deps in https://github.com/Automattic/syntaxhighlighter/pull/269

### CODE REVIEW

File: syntaxhighlighter.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 Text Domain:  syntaxhighlighter
 License:      GPL2
 License URI:  https://www.gnu.org/licenses/gpl-2.0.html
-Requires at least: 5.7
+Requires at least: 6.1
```
Comment (fjorgemota) [inline syntaxhighlighter.php]: This aligns the minimum WP version supported with the one supported by WPJM.

Given that 6.1 is already ~1 year old, and 5.7 is almost ~3 (!!!) years old to this point, I think it is safe to ask people to upgrade. :)

**Comment by fjorgemota** [comment]
This is the ZIP file I built based on this PR, by the way: 

[syntaxhighlighter.zip](https://github.com/Automattic/syntaxhighlighter/files/13552397/syntaxhighlighter.zip)

File: syntaxhighlighter.php (hunk 1 above)
Comment (thedebian) [reply syntaxhighlighter.php]: If it works with 5.7, let's keep it that way (if it doesn't, then, fine :D).
We won't necessarily make any breaking change in the near (or far) future, so if this release doesn't break compat with 5.7, we should keep it as-is.

**Review by thedebian — APPROVED** [review]
🚀

File: syntaxhighlighter.php (hunk 1 above)
Comment (fjorgemota) [reply syntaxhighlighter.php]: Fair enough, I tested it on PHP 7.0 and WP 5.7 and reverted the commit in 9f582dc. Thanks!

**Pull request merged**